packages/react-core/src/demos/ComposableMenu/examples/ComposableMultipleTypeaheadSelect.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import {
+  Menu,
+  MenuContent,
+  MenuList,
+  MenuItem,
+  Popper,
+  TextInputGroup,
+  TextInputGroupMain,
+  TextInputGroupUtilities,
+  ChipGroup,
+  Chip,
+  Button
+} from '@patternfly/react-core';
+import TimesIcon from '@patternfly/react-icons/dist/esm/icons/times-icon';
+import CaretDownIcon from '@patternfly/react-icons/dist/esm/icons/caret-down-icon';
+
+export const ComposableMultipleTypeaheadSelect: React.FunctionComponent = () => {
+  const [isOpen, setIsOpen] = React.useState<boolean>(false);
+  const [selected, setSelected] = React.useState<string[]>([]);
+  const [inputValue, setInputValue] = React.useState<string>('');
+  const toggleRef = React.useRef<HTMLDivElement>();
+  const inputRef = React.useRef<HTMLInputElement>();
+  const menuRef = React.useRef<HTMLDivElement>();
+  const containerRef = React.useRef<HTMLDivElement>();
+
+  const options = [
+    <MenuItem key={0} itemId="Option 1">
```

Comment:
Can we update the data a little her so we can get varying results when we start typing?  Right now you only get matches on all items or one.